searx/result_types/_base.py

```diff
@@ -446,7 +446,7 @@ class LegacyResult(dict):
     positions: list[int]
     score: float
     category: str
-    publishedDate: datetime.datetime | None = None
+    publishedDate: datetime.datetime | None
```

Comment:
I guess the default here is not the problem / this modification is not needed (compare with `MainResult.publishedDate`).

Reply:
your are right / forget my comment, I mixed how `MainResult` (a class) and `LegacyResult` ( a `dict`) do work .. :facepalm: